src/pages/Explore/index.tsx

```diff
@@ -12,24 +13,33 @@ const ExploreContainer = styled.div`
 const TokenTableContainer = styled.div`
   padding: 16px 0px;
 `
+const TitleContainer = styled.div`
+  font-size: 32px;
+  margin-bottom: 16px;
+  max-width: 960px;
+  margin-left: auto;
```

Comment:
`margin: 0 auto`? although i feel like usually you don't need that if its a flex container